### PR TITLE
feat: Webpack updated to version 5

### PR DESCRIPTION
**Pull request by luisfelipefurlan**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
feature

* **What is the current behavior?** (You can also link to an open issue here)
the behavior has not changed

* **What is the new behavior (if this is a feature change)?**
the behavior has not changed

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
dojot/dojot#2443

* **Other information**:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -209,7 +212,7 @@
   },
   "husky": {
```
Comment (LuanEdCosta) [inline package.json:213]: I believe that we don't need the husky config to be in package.json file anymore. It already is on  `.husky/` folder in the root directory.

**Review by LuanEdCosta — CHANGES_REQUESTED** [review]
Doesn't make sense to delete important lines from saga tests because the tests are failing. We should let tests untouched and find a better solution.

Maybe we can open an issue on `redux-saga-test-plan`, downgrade some babel dependencies versions or something else, but these tests are important for the project, delete test lines should be the last solution.

File: package.json (hunk 1 above)
Comment (luisfelipefurlan) [reply package.json:213]: in fact it is no longer necessary, that was a leftover.

**Review by LuanEdCosta — DISMISSED** [review]
I will approve even with a test getting a timeout warning. We can fix it later.

**Review by LuanEdCosta — APPROVED** [review]

**Pull request merged**